src/views/mainView.jsx

```diff
@@ -1,4 +1,5 @@
-import React from "react";
+import React, {useEffect} from "react";
```

Comment:
why is this needed?

Reply:
I will remove it was doing some testing using useEffect